dts/common/espressif/partitions_default_8M.dtsi

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2024 Espressif Systems (Shanghai) Co., Ltd.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+ /* 8MB flash partition table */
+&flash0 {
+	partitions {
+		compatible = "fixed-partitions";
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+		/* Bootloader: 64kB */
+		boot_partition: partition@0 {
+			label = "mcuboot";
+			reg = <0x0 0x10000>;
+		};
+
+		/* Application storage: default 64kB, adjustable to 192kB */
+		storage_partition: partition@10000 {
+			label = "storage";
+			reg = <0x10000 0x10000>;
+		};
+
+		/* Slot0: 3904kB */
+		slot0_partition: partition@40000 {
+			label = "image-0";
+			reg = <0x40000 0x3d0000>;
+		};
+
+		/* Slot1: 3904kB */
+		slot1_partition: partition@410000 {
+			label = "image-1";
+			reg = <0x410000 0x3d0000>;
+		};
+
+		/* Optional scratch */
+		scratch_partition: partition@7e0000 {
```

Comment:
I think scratch partition could be removed from everywhere.

Reply:
I know it is not used anymore in MCUboot swap, but there are a few samples that still use the `scratch_partition` so I would rather leave it there.